packages/hash/playwright/tests/entity-creation.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { loginUsingTempForm } from "./shared/login-using-temp-form";
+import { resetDb } from "./shared/reset-db";
+import { expect, Locator, Page, test } from "./shared/runtime";
+
+const getCellText = async (
+  canvasLocator: Locator,
```

Comment:
Playwright docs don’t add `Locator` suffix to variable names. You refer to canvas as `canvas` in the test itself, we could do so in the helper functions too:

```suggestion
  canvas: Locator,
```

Reply:
Updated here https://github.com/hashintel/hash/pull/1790/commits/29548ceb30f92b07c05ef2c19cce6daf2ad59f94